packages/actor-init-sparql/test/HttpServiceSparqlEndpoint-test.ts

```diff
@@ -511,6 +511,7 @@ describe('HttpServiceSparqlEndpoint', () => {
 
         expect(instance.writeQueryResult)
           .toHaveBeenCalledWith(engine, stdout, stderr, request, response, 'test_parseBody_result', chosen, false);
+        jest.dontMock('negotiate');
```

Comment:
The end of a test feels like a strange place for this.
Do things still work properly when executing tests separately?
If not, then I would suggest moving this to the `beforeEach` of the `determineMediaType` block.

Reply:
They do work properly separately. The mock is only used in that one test. So i thought it would be more efficient to turn it off in the same test. But if preferred I can put it in the determineMediatype.

Reply:
Ah, I missed the `doMock` earlier in that test. Never mind my comment 🙂